### PR TITLE
Updated 2 redirects for www

### PR DESCRIPTION
**Pull request by kathycarothers**
FY reports for 2016 and 2017

### CODE REVIEW

File: redirects-www.conf [hunk 1]
```diff
@@ -330,6 +330,8 @@ rewrite ^/resources/about-fec/reports/gifts-to-foreigners-fy2016-letter.pdf http
 
 # Redirects for /resources/cms-content/documents/
 rewrite ^/resources/cms-content/documents/AllPublicFunds.pdf https://www.fec.gov/resources/cms-content/documents/AllPublicFunds.xls redirect;
+rewrite ^/resources/cms-content/documents/FY2016_AFR.pdf https://www.fec.gov/resources/cms-content/documents/FY2016_AFR.pdf redirect;
+rewrite ^/resources/cms-content/documents/fy_2017_Congressional_budget.pdf https://www.fec.gov/resources/cms-content/documents/fy_2017_Congressional_budget.pdf redirect;
```
Comment (patphongs) [inline redirects-www.conf:334]: @kathycarothers These don't look right to me, it seems to be a www redirect that is redirecting back to itself? What's the intent of this redirect?

**Review by patphongs — CHANGES_REQUESTED** [review]

**Comment by kathycarothers** [comment]
I have updated the first part of the code that was incorrect. I should of doubled checked that.

**Review by patphongs — APPROVED** [review]
Thanks for making those corrections @kathycarothers, looks good!

**Review by dorothyyeager — APPROVED** [review]
Looks good.

**Pull request merged**